tests/sysprep_test.go

```diff
@@ -0,0 +1,361 @@
+    <ComputerName>ANAME-PC</ComputerName>
+    <ProductKey>%s</ProductKey>
+    </component>
+    </settings>
+    <settings pass="oobeSystem">
+    <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <AutoLogon>
+    <Password>
+    <Value>Gauranga</Value>
+    <PlainText>true</PlainText>
+    </Password>
+    <Enabled>false</Enabled>
+    <Username>admin</Username>
+    </AutoLogon>
+    <OOBE>
+    <HideEULAPage>true</HideEULAPage>
+    <HideOEMRegistrationScreen>true</HideOEMRegistrationScreen>
+    <HideOnlineAccountScreens>true</HideOnlineAccountScreens>
+    <HideWirelessSetupInOOBE>true</HideWirelessSetupInOOBE>
+    <NetworkLocation>Work</NetworkLocation>
+    <ProtectYourPC>1</ProtectYourPC>
+    <SkipUserOOBE>true</SkipUserOOBE>
+    <SkipMachineOOBE>true</SkipMachineOOBE>
+    </OOBE>
+    <UserAccounts>
+    <LocalAccounts>
+    <LocalAccount wcm:action="add">
+    <Password>
+    <Value>Gauranga</Value>
+    <PlainText>true</PlainText>
+    </Password>
+    <Description></Description>
+    <DisplayName>admin</DisplayName>
+    <Group>Administrators</Group>
+    <Name>admin</Name>
+    </LocalAccount>
+    </LocalAccounts>
+    </UserAccounts>
+    <RegisteredOrganization></RegisteredOrganization>
+    <RegisteredOwner>admin</RegisteredOwner>
+    <DisableAutoDaylightTimeSet>false</DisableAutoDaylightTimeSet>
+    <TimeZone>AUS Eastern Standard Time</TimeZone>
+    <VisualEffects>
+    <SystemDefaultBackgroundColor>2</SystemDefaultBackgroundColor>
+    </VisualEffects>
+    </component>
+    <component name="Microsoft-Windows-ehome-reg-inf" processorArchitecture="x86" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="NonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <RestartEnabled>true</RestartEnabled>
+    </component>
+    <component name="Microsoft-Windows-ehome-reg-inf" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="NonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <RestartEnabled>true</RestartEnabled>
+    </component>
+    </settings>
+    </unattend>
+    `
+)
+
+func insertProductKeyToAnswerFileTemplate(answerFileTemplate string) string {
+	productKeyFilePath := os.Getenv("KUBEVIRT_WINDOWS_PRODUCT_KEY_PATH")
+	keyFromFile, err := ioutil.ReadFile(productKeyFilePath)
+	Expect(err).ToNot(HaveOccurred())
+	productKey := strings.TrimSpace(string(keyFromFile))
+	return fmt.Sprintf(answerFileTemplate, productKey)
+}
+
+var getWindowsSysprepVMISpec = func() v1.VirtualMachineInstanceSpec {
```

Comment:
@borod108 I've noticed the same function in `windows_test.go`, and I wonder why don't you aggregate the sysprep test into the windows file?

Reply:
This was my reasoning: for this test we are using a different image, different by nature since it is a sealed image not just by version and size also this test is to run in a separate lane, nightly not per push.
I started with putting it in the same file, but I figure that putting this in a separate file will make it clear that this is a whole separate thing, and will not force a person that is dealing with the windows tests to also deal with sysprep related problems and some minor details that might differ when running different window versions as far as I can see this will not improve test coverage but may make the test maintenance harder.
But if you insist I can refactor this.